conf/routes

```diff
@@ -49,6 +49,7 @@ GET           /streams/:stream_id/outputs
 GET           /streams/:stream_id/alerts                                                         @controllers.AlertsController.index(stream_id: String)
 POST          /streams/:stream_id/alerts/conditions/add/messagecount                             @controllers.AlertsController.addTypeMessageCount(stream_id: String)
 POST          /streams/:stream_id/alerts/conditions/add/fieldvalue                               @controllers.AlertsController.addTypeFieldValue(stream_id: String)
+POST          /streams/:stream_id/alerts/conditions/add/fielstringdvalue                         @controllers.AlertsController.addTypeFieldStringValue(stream_id: String)
```

Comment:
haha, good catch!

![](http://www.reactiongifs.com/r/gcom.gif)

fixed in bbd0e2e :)